### DISCUSSION TITLE
BUG: Issues with <easyclick_all> or <gridmove> with multiple screens

### DISCUSSION THREAD

**Discussion by Dawid-Deszcz** [pit-ray/win-vind#234] [Development]
Hi guys and galls, I have just started using win-vind so appologise if this problem should be described somewhere else, I also searched the forum and it appears that I am the first person who have noticed this issue.

**BUG DESCRITPION**
When I choose <easyclick_all> or <gridmove>, I have both options working perfectly on my primary screen but on my secondary screen it shows up from time to time it is especially bad if chrome is opened on the secondary screen. 

The problem is with letter-shortucts not being visible on the secondary screen. Evertyhing works but I have to remember what was a letter shortucs in a particular place before. If I know where certain shortuct is e.g. SSX and press it then my mouse will move to this location, but it is impossible to work that way because I have to "guess" or remember were certain shortucs usually are and how they are named (with <gridmove> they usualy are in the same place every time) 

**BUG WORKAROUND**
If I choose <easyclick_all> or <gridmove>, I can move my mouse around on my secondary screen and options will show up especially at placys were mouse was (but I have to touch the mouse not the keyboard -  I do not want to use my mouse at all, using win-vind mouse movements does not have the same effect).

**Comment by Dawid-Deszcz** [comment]
**Update**
It is clearly connected with win-vind generated text visibility on my second screen.

I just realised that any text generated by win-vind is sometimes not visible on my second screen it is not consistent, sometimes it works sometimes it does not. It usually does not work if my Chrome or Edge are in maximized. When it does not work it looks as if the text was "hidden" in the background (behind chrome).

Please let me know if somebody has experienced simillar issues.

**Comment by qaptoR** [comment]
I'm bumping this because I'm just getting back into win-vind. I have 4 monitors, and grid-move only works for 2 of the monitors.
The main monitor looks fine, the second monitor, the labels flicker like crazy (but it still works), and then for monitor 3 and 4 (honestly the two I use the most, 1 and 2 are mostly for reference documents) there are no labels at all.

**Reply by shrisheshpratik** [reply]
### Workaround 1:
Keep monitors in horizontal line

![image](https://github.com/user-attachments/assets/00c90fd4-aeba-496b-8429-e647d670cf9b)

This will generate gridmove in all the monitors.

### Workaround 2:
In GUI Normal Mode ( `Ctrl + ]` )
Use [`CtrlW+s`] and select the application using [`Tab/ Shift Tab`]
Press key like [`Space`] to focus the cursor to new application window
Then press `f` in gui normal to move around screen by keyboard layout

### Workaround 3:
In GUI Normal Mode ( `Ctrl + ]`  )
Use [`CtrlW+(h/j/k/l)`] to move cursor across different screens
Then press `f` in gui normal to move around screen by keyboard layout

**Note:** You can always try using <easyclick_all> `[A (Case sensitive) + o(left click)/a (right click)]` in gui normal  mode and see if it is pointing to correct clickable elements.

@pit-ray If possible please address this issue of handling gridmove if monitor alignments are not all in same horizontal level like:
![image](https://github.com/user-attachments/assets/e6b85cc2-5ea7-40e9-8ad4-86a204543205)

Honestly this is the only issue which I am facing otherwise win-vind has made my windows experience so F*king better, I will always be in debt
